casper/src/main/scala/coop/rchain/casper/util/rholang/RuntimeManager.scala

```diff
@@ -38,10 +42,7 @@ class RuntimeManager private (val emptyStateHash: ByteString, runtimeContainer:
     //TODO: Is better error handling needed here?
     val result: Seq[Datum[ListParWithRandom]] =
       if (processedDeploy.status.isFailed) Nil
-      else {
-        val returnChannel = Par().copy(exprs = Seq(Expr(GString(name))))
-        runtime.space.getData(returnChannel)
-      }
+      else runtime.space.getData(name)
```

Comment:
Does this mean I can provide the RuntimeManager with fake/invalid/additional results by sending  to the `@"__SCALA__"` channel?

Reply:
This would work but the pattern is used only in tests where you start with fresh Tuplespace state, so you would need to do it by yourself and purposefully. You can't mess with the result of the possible future tests. This pattern though will disappear though soon when we disallow sending on forgeable names.